lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -147,6 +147,15 @@
   groups:
     description:
       - List of additional group names and/or IDs that the container process will run as.
+  healthcheck:
+    type: dict
+    description:
+      - Configure a check that's run to determine whether container is healthy
+      - 'I(test) - Test to perform to determine container health.'
```

Comment:
You should document the elements in the dict better. It is not clear from this short description that `test` is a list of strings, which is the command which will be executed (and what `CMD` in front of it does). Also, it is unclear to me how this command reports the health (return code? timeout?). If it is too complicated to describe it here, you should link to a document where this is explained in detail (docker documentation?) and say that it is similar to there.

Reply:
In general: you should also document default values, if there are any.